### PR TITLE
feat: [CDS-37779]: added 'open in new tab' option for services list

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Jira Links:

https://harness.atlassian.net/browse/CDS-37779

##### Screenshots:

https://user-images.githubusercontent.com/98325173/173550933-22781019-61ee-4483-9289-c57647719566.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

File: src/modules/75-cd/components/Services/ServicesList/ServicesList.tsx [hunk 1]
```diff
@@ -432,6 +450,15 @@ const RenderColumnMenu: Renderer<CellProps<any>> = ({ row, column }) => {
...
         <Menu style={{ minWidth: 'unset' }}>
+          <Link
+            className={cx('bp3-menu-item', css.openNewTabStyle)}
+            target="_blank"
+            to={openInNewTab}
+            onClick={e => e.stopPropagation()}
+          >
+            <Icon name="launch" style={{ marginRight: '5px' }} />
+            {getString('pipeline.openInNewTab')}
+          </Link>
```
[the remark this answers is not in the document]
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/Services/ServicesList/ServicesList.tsx:461]: sure will add test case for this

**Review by kapilkumawat86 — APPROVED** [review]

File: src/modules/75-cd/components/Services/ServicesList/ServicesList.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/Services/ServicesList/ServicesList.tsx:461]: Made this ticket regarding the test cases - https://harness.atlassian.net/browse/PIE-4089
Thanks

**Pull request merged**